FRAuth/FRAuth/DeviceBinding/UserDeviceKeyService.swift

```diff
@@ -0,0 +1,90 @@
+// 
+//  UserDeviceKeyService.swift
+//  FRAuth
+//
+//  Copyright (c) 2022 ForgeRock. All rights reserved.
+//
+//  This software may be modified and distributed under the terms
+//  of the MIT license. See the LICENSE file for details.
+//
+
+
+import Foundation
+
+
+public protocol UserKeyService {
+    
+    /// Fetch the key existence status in device.
+    /// - Parameter  userId: optional and received from server
+    func getKeyStatus(userId: String?) -> KeyFoundStatus
+    
+    
+    /// Get all the user keys in device.
+    var userKeys: [UserKey] { get set }
+}
+
+
+internal class UserDeviceKeyService: UserKeyService {
```

Comment:
please align this whole class

Reply:
It is actually aligned. There are just couple of nested indentations